sw/device/tests/uart_parity_test.c

```diff
@@ -0,0 +1,211 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_base.h"
+#include "sw/device/lib/dif/dif_clkmgr.h"
+#include "sw/device/lib/dif/dif_lc_ctrl.h"
```

Comment:
These two headers are not used and can be dropped.
Please also check the Bazel file.

Reply:
Thanks, this test was originally part of `uart_tx_rx` and I forgot to remove these.